modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -143,18 +151,39 @@ export default {
     this.modal.active = true;
 
     if (this.docId) {
-      // TODO: Get data here.
+      let docData;
+      try {
+        const getOnePath = `${this.moduleOptions.action}/${this.docId}`;
+        docData = await apos.http.get(getOnePath, {
+          busy: true
+        });
+      } finally {
```

Comment:
Does `finally` make sense here? Can it really cope if `this.doc.data` is undefined? It is good not to just get wedged, it's true. Maybe we need a catch block too to display an error, if that's a thing we're ready to do?

Reply:
There should be a catch. We'd probably want to deliver an error an then exit out of the modal.